tests/test_class_contracts.py

```diff
@@ -217,6 +217,27 @@ def test_circle_area_invalid() -> None:
     assert "r > 0" in msg
 
 
+def test_pizza_valid() -> None:
+    """
+    Test the Pizza representation invariant on a valid instance.
+    """
+    pizza = Pizza(radius=5, ingredients=["cheese", "pepperoni"])
+    assert pizza.radius == 5 and pizza.ingredients == ["cheese", "pepperoni"]
+
+
+def test_pizza_invalid() -> None:
+    """
+    Test the Pizza representation invariant on an invalid instance.
+    """
+    with pytest.raises(AssertionError) as excinfo:
+        Pizza(radius=10, ingredients=[])
+    msg = str(excinfo.value)
+    assert (
+        "\"len(self.ingredients) > 0\" was violated for arguments {'radius': 10, 'ingredients': []}"
+        in msg
```

Comment:
You shouldn't use the `in` operator, you can check for exact string equality

Reply:
I noticed that the `in` operator is used in a few of the old tests as well, should I change them all to use `==` instead?

Reply:
@yibinglu the other tests use `in` because they're just checking that the correct assertion expression is in the message. But in this case, you're writing out the full message, so `==` is more appropriate.